### ISSUE TITLE
 Converting simplified .csv files to traditional .csv files #3 

### ISSUE THREAD

**Issue by nh36** [hanproj/received-shi#3]
The files

1 Pre Qin Poetry.csv
2 Qin Poetry.csv
3 Han Poetry.csv
4 Nanbeichao Poetry.csv
Are currently in simplified characters and they should be in traditional characters. This conversion, I expect will be easy, e.g. using SinoPy

**Comment by LinguList** [comment]
I suggest to do this when working with the CLDF conversion, not before 
doing it. There it can be easily done with sinopy.

**Comment by nh36** [comment]
@ashhenson I think this one is also maybe superceded, I am not really sure. I think maybe it is worth checking still to see whether there are shi in these .csv files that are not in Lu. 

Also, maybe after you get the community annotator up and running, if we don't feel happy with the results we can try adding the Qin and Nanbeichao shi at it. In any case, something to think about. But for now, let's try to just keep these issues updated to the current state of plan (doesn't mean doing them, just making sure that if they are still open they are worded correctly.)

**Comment by ashhenson** [comment]
@nh36 Yeah, this issue is still open. It kinda fell by the wayside in the run up to Vienna. The plan is to remove all punctuation and run string correlations (that way, they don't have to be exactly same -- they may have different variants, for instance -- and we can still tell which ones are really similar).
There are a lot of poems either with no name or with the same name (in the Lu 1983 data, the same author has different poems by the same name).
